core/market/src/negotiation/common.rs

```diff
@@ -417,12 +404,14 @@ impl CommonBroker {
                 RemoteAgreementError::InternalError(agreement_id.clone())
             })?;
 
+        self.notify_agreement(&agreement).await;
+
         match terminator {
             OwnerType::Provider => counter!("market.agreements.provider.terminated", 1),
             OwnerType::Requestor => counter!("market.agreements.requestor.terminated", 1),
         };
 
-        terminate_reason_metric(&msg.reason, owner_type);
+        inc_terminate_metrics(&msg.reason, owner_type);
```

Comment:
this now contains the above metrics, so we need to remove them :(
sorry, my fault